code/game/gamemodes/cult/runes.dm

```diff
@@ -139,6 +139,8 @@ fail_invoke() is called when the rune fails, via not enough people around or oth
 structure_check() searches for nearby cultist structures required for the invocation. Proper structures are pylons, forges, archives, and altars.
 */
 /obj/effect/rune/proc/can_invoke(mob/living/user)
```

Comment:
I think it'd be more intuitive if you also couldn't draw the runes in the first place. Might be frustrating.

Also, these runes badly suffer from not telling people what is wrong with them. This might be a good opportunity to add a simple bit of feedback for a fizzing out rune.

Reply:
I tried to follow a design principle of, if it is a mundane action it still works. You can still draw a rune, you can still cut yourself, you can still use a spear as a spear. That said, you may have a point here, I wouldn't mind changing it so runes can't be drawn at all. Open for discussion here!

Reply:
Maybe a bit late, but I agree with Flattest on this one. Someone could theoretically set up a whole base in the chapel office only for none of the runes to work when they actually need them.